test/test_app_details.py

```diff
@@ -96,20 +101,21 @@ def test_get_app_base_url_returns_empty_string_when_host_cannot_be_determined(mo
 
 @patch(GET_MANIFEST_JSON, return_value=manifest_path('installed_no_console_ip.json'))
```

Comment:
None of the tests for get_app_base_url or q_url_for should need a patch for GET_MANIFEST_JSON. If there are any manifest files in the test/manifests directory that are no longer needed, delete them.